From cad7603ccdc612a444a950869f3583088b30df33 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Sat, 27 Jun 2020 18:43:17 +0000
Subject: [PATCH] Qt: unbreak build without SDL

Qt/QtMain.cpp:51:1: error: unknown type name 'SDL_AudioSpec'
SDL_AudioSpec g_retFmt;
^
1 error generated.
---
 Qt/QtMain.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Qt/QtMain.cpp b/Qt/QtMain.cpp
index 7713b6587d..7f9726b417 100644
--- a/Qt/QtMain.cpp
+++ b/Qt/QtMain.cpp
@@ -48,9 +48,9 @@ static int browseFileEvent = -1;
 static int browseFolderEvent = -1;
 QTCamera *qtcamera = nullptr;
 
+#ifdef SDL
 SDL_AudioSpec g_retFmt;
 
-#ifdef SDL
 static SDL_AudioDeviceID audioDev = 0;
 
 extern void mixaudio(void *userdata, Uint8 *stream, int len) {
@@ -147,10 +147,12 @@ std::string System_GetProperty(SystemProperty prop) {
 
 int System_GetPropertyInt(SystemProperty prop) {
 	switch (prop) {
+#if defined(SDL)
 	case SYSPROP_AUDIO_SAMPLE_RATE:
 		return g_retFmt.freq;
 	case SYSPROP_AUDIO_FRAMES_PER_BUFFER:
 		return g_retFmt.samples;
+#endif
 	case SYSPROP_DEVICE_TYPE:
 #if defined(__ANDROID__)
 		return DEVICE_TYPE_MOBILE;
